kolibri/plugins/coach/assets/src/state/actions/exam.js

```diff
@@ -40,11 +41,57 @@ function _groupsState(groups) {
   return groups.map(group => _groupState(group));
 }
 
+function _breadcrumbState(topic) {
+  return {
+    id: topic.pk,
+    title: topic.title,
+  };
+}
+
+
+function _breadcrumbsState(topics) {
+  return topics.map(topic => _breadcrumbState(topic));
```

Comment:
Since this is the only place _breadcrumbState is used, I would consider just inlining it here.